### PR TITLE
4 packages from AestheticIntegration/ocaml-opentelemetry at 0.2

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`opentelemetry.0.2`: Instrumentation for https://opentelemetry.io
-`opentelemetry-client-ocurl.0.2`: Collector client for opentelemetry, using http + ocurl
-`opentelemetry-cohttp-lwt.0.2`: Opentelemetry tracing for Cohttp HTTP servers
-`opentelemetry-lwt.0.2`: Lwt-compatible instrumentation for https://opentelemetry.io

---
* Homepage: https://github.com/aestheticintegration/ocaml-opentelemetry
* Source repo: git+https://github.com/aestheticintegration/ocaml-opentelemetry.git
* Bug tracker: https://github.com/aestheticintegration/ocaml-opentelemetry/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/opentelemetry-cohttp-lwt/opentelemetry-cohttp-lwt.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+homepage: "https://github.com/aestheticintegration/ocaml-opentelemetry"
+bug-reports:
+  "https://github.com/aestheticintegration/ocaml-opentelemetry/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "opentelemetry" {= version}
+  "opentelemetry-lwt" {= version}
+  "odoc" {with-doc}
+  "cohttp-lwt"
```
Comment (mseri) [inline packages/opentelemetry-cohttp-lwt/opentelemetry-cohttp-lwt.0.2/opam:16]:
```suggestion
  "cohttp-lwt" {>= "5.0.0"}
```
for
```
       ocamlc src/integrations/cohttp/.opentelemetry_cohttp_lwt.objs/byte/opentelemetry_cohttp_lwt.{cmi,cmo,cmt} (exit 2)
- (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -warn-error -a+8 -g -bin-annot -I src/integrations/cohttp/.opentelemetry_cohttp_lwt.objs/byte -I /home/opam/.opam/4.14/lib/base -I /home/opam/.opam/4.14/lib/base/base_internalhash_types -I /home/opam/.opam/4.14/lib/base/caml -I /home/opam/.opam/4.14/lib/base/shadow_stdlib -I /home/opam/.opam/4.14/lib/base64 -I /home/opam/.opam/4.14/lib/bytes -I /home/opam/.opam/4.14/lib/cohttp -I /home/opam/.opam/4.14/lib/cohttp-lwt -I /home/opam/.opam/4.14/lib/fieldslib -I /home/opam/.opam/4.14/lib/lwt -I /home/opam/.opam/4.14/lib/ocaml/threads -I /home/opam/.opam/4.14/lib/opentelemetry -I /home/opam/.opam/4.14/lib/opentelemetry-lwt -I /home/opam/.opam/4.14/lib/pbrt -I /home/opam/.opam/4.14/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.14/lib/ptime -I /home/opam/.opam/4.14/lib/ptime/os -I /home/opam/.opam/4.14/lib/re -I /home/opam/.opam/4.14/lib/re/posix -I /home/opam/.opam/4.14/lib/result -I /home/opam/.opam/4.14/lib/seq -I /home/opam/.opam/4.14/lib/sexplib0 -I /home/opam/.opam/4.14/lib/stringext -I /home/opam/.opam/4.14/lib/uri -I /home/opam/.opam/4.14/lib/uri/sexp -no-alias-deps -o src/integrations/cohttp/.opentelemetry_cohttp_lwt.objs/byte/opentelemetry_cohttp_lwt.cmo -c -impl src/integrations/cohttp/opentelemetry_cohttp_lwt.ml)
- File "src/integrations/cohttp/opentelemetry_cohttp_lwt.ml", line 231, characters 10-16:
- 231 |   (module Traced : Cohttp_lwt.S.Client)
-                 ^^^^^^
- Error: Signature mismatch:
-        ...
-        The value `sexp_of_ctx' is required but not provided
-        File "cohttp-lwt/src/s.ml", line 30, characters 2-31:
-          Expected declaration
-        The value `default_ctx' is required but not provided
-        File "cohttp-lwt/src/s.ml", line 31, characters 2-23:
-          Expected declaration
```

File: packages/opentelemetry-cohttp-lwt/opentelemetry-cohttp-lwt.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+homepage: "https://github.com/aestheticintegration/ocaml-opentelemetry"
+bug-reports:
+  "https://github.com/aestheticintegration/ocaml-opentelemetry/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "opentelemetry" {= version}
+  "opentelemetry-lwt" {= version}
+  "odoc" {with-doc}
+  "cohttp-lwt" {>= "5.0.0"}
```
Comment (mseri) [inline packages/opentelemetry-cohttp-lwt/opentelemetry-cohttp-lwt.0.2/opam:16]:
```suggestion
  "cohttp-lwt" {>= "4.0.0"}
```
I think this should work also with 4.0.0, let’s try

**Comment by mseri** [comment]
Thanks. Can you upstream the updated lower bound?

**Comment by c-cube** [comment]
done!

**Comment by c-cube** [comment]
thank you :grin:

**Pull request merged**